druntime/test/exceptions/Makefile

```diff
@@ -1,7 +1,19 @@
-TESTS=stderr_msg unittest_assert invalid_memory_operation unknown_gc static_dtor \
+ifeq ($(OS),linux)
+    # Alpine Linux comes with an apk tool
+    ifeq (1,$(shell which apk &>/dev/null && echo 1))
+        IS_MUSL:=1
+    endif
+endif
+
+TESTS=stderr_msg unittest_assert invalid_memory_operation static_dtor \
       future_message refcounted rt_trap_exceptions_drt catch_in_finally \
       message_with_null
 
+# FIXME: segfaults with musl libc
+ifneq ($(IS_MUSL),1)
+TESTS += unknown_gc
+endif
```

Comment:
This was unexpected, works with LDC (and its druntime v2.110 though). The test looks as harmless as it gets.